### ISSUE TITLE
[Bug]: Update device properties error upgrade to eufy-security-ws Container 1.3.2

### ISSUE THREAD

**Issue by patrickli** [bropat/eufy-security-client#305]
### Client version

2.4.2

### Node version

16.19.1

### Operating System type

Docker

### Operating system version

eufy-security-ws Container 1.3.3

### Describe the bug

The following error message starts to appear after upgrading the container. The device is an Eufy Floodlight Cam E 2K T8422.

The server and integration to Home Assistant for the device seems to be working fine. I can control the light and start/stop P2P streaming.

### To reproduce

Just by starting the server.

### Screenshots & Logfiles

```
eufy-security-ws  | 2023-02-25 01:58:00.698  INFO  Eufy Security server listening on host 0.0.0.0, port 3000
eufy-security-ws  | 2023-02-25 01:58:05.394  INFO  Connected to station T8422xxxxxxxx on host xx.xx.xx.xx and port 21505
eufy-security-ws  | 2023-02-25 01:58:05.617  ERROR Update device T8422xxxxxxxx properties error
eufy-security-ws  |  DeviceNotFoundError  Device with this serial T8422xxxxxxxx doesn't exists!
eufy-security-ws  | error stack:
eufy-security-ws  | • eufysecurity.ts:354 getDevice
eufy-security-ws  |     node_modules/eufy-security-client/src/eufysecurity.ts:354:15
eufy-security-ws  | • eufysecurity.ts:719 updateDeviceProperties
eufy-security-ws  |     node_modules/eufy-security-client/src/eufysecurity.ts:719:14
eufy-security-ws  | • eufysecurity.ts:431 <anonymous>
eufy-security-ws  |     node_modules/eufy-security-client/src/eufysecurity.ts:431:113
eufy-security-ws  | • node:events:513 emit
eufy-security-ws  |     node:events:513:28
eufy-security-ws  | • station.ts:827 <anonymous>
eufy-security-ws  |     node_modules/eufy-security-client/src/http/station.ts:827:18
eufy-security-ws  | • station.ts:826 onCameraInfo
eufy-security-ws  |     node_modules/eufy-security-client/src/http/station.ts:826:30
eufy-security-ws  | • station.ts:59 <anonymous>
eufy-security-ws  |     node_modules/eufy-security-client/src/http/station.ts:59:87
eufy-security-ws  | • node:events:513 emit
eufy-security-ws  |     node:events:513:28
eufy-security-ws  | • session.ts:1481 handleDataControl
eufy-security-ws  |     node_modules/eufy-security-client/src/p2p/session.ts:1481:30
eufy-security-ws  | 2023-02-25 01:58:15.642  INFO  Push notification connection successfully established
```

**Comment by bropat** [comment]
@patrickli 

With the latest optimizations, the P2P connection can be established very quickly and it can happen that the data received from the P2P connection tries to update some properties of the connected devices, even though they have not been created/initialized yet.
This error has no negative impact at the moment, apart from logging the error.
Other work has more priority at the moment.

**Comment by patrickli** [comment]
Alright. Thanks for the explanation.

**Comment by bropat** [comment]
Fixed in version `2.4.4`.
